lib/helpers.js

```diff
@@ -1,5 +1,7 @@
 'use babel';
 
+/* eslint no-underscore-dangle: ["error", { "allow": ["_postcssResult"] }] */
```

Comment:
Use `// eslint-disable-next-line no-underscore-dangle` right before the line that is triggering this instead of setting it file wide.